index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="css/icons.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <title>Portfolio of Michael Mailman</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-title">
+        <h1>Michael Mailman</h1>
+      </div>
+      <nav>
+        <div class="icon-menu">
+        </div>
+        <ul>
+          <li><a href="#about-me">About Me</a></li>
+          <li><a href="#class-201">Class 201 Projects</a></li>
+          <li><a href="#class-301">Class 301 Projects</a></li>
+          <li><a href="http://www.linkedin.com/in/mailmanmichael" class="icon-linkedin">LinkedIn</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section id="about-me">
+        <h3>About Me</h3>
+        <img src="imgs/mailman.jpeg" alt="BAD PHOTO" />
```

Comment:
I'm not going to say one way or another, but for a portfolio you should try to find a professional picture of yourself you like.